imp_services/displayLogs.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Created by Trevor on 7/3/2015.
+ */
+var Q = require("q");
+
+/* var a = {};
+ a["key1"] = "value1";
+ a["key2"] = "value2";
+ */
+
+var LogTypeMap = {};
+LogTypeMap[100] = "Added Pile";
+LogTypeMap[200] = "Added Run";
+LogTypeMap[300] = "Removed Pile";
+LogTypeMap[400] = "Removed Run";
+LogTypeMap[500] = "Audited";
+LogTypeMap[600] = "Noted";
+LogTypeMap[700] = "Created New Product";
+LogTypeMap[800] = "Created User";
+
+var functionMap = {};
+functionMap[100] = toStringDefault;
+functionMap[200] = toStringDefault;
+functionMap[300] = toStringDefault;
+functionMap[400] = toStringDefault;
+functionMap[500] = toStringDefault;
+functionMap[600] = toStringDefault;
+functionMap[700] = toStringDefault;
+functionMap[800] = toStringDefault;
+
+var stringLogs = [];
+var jsonString = '{"logs":[';
+
+function toStringDefault (jsonObj) {
+    var logObj = JSON.parse(jsonObj);
+    var actionPiece = JSON.parse(logObj.action);
+    console.log("Action Piece " + actionPiece.value);
+    return logObj.username + " on " + logObj.time + " " + LogTypeMap[logObj.logType] + " " + actionPiece.value;
+}
+
+module.exports =
+
```

Comment:
Can you please add a utility function:
_verifyKey = function(key){
LogMap[key]? return true: return false;
}